conf/application.conf

```diff
@@ -233,6 +234,10 @@ minimum-tax-year.version-2 {
   historic = 2017
 }
```

Comment:
Check if these version numbers are accurate (why are we using version 2 in v5 code?)